### PR TITLE
only for testing reviews

### PR DESCRIPTION
**Pull request by AviralVerma13**
- Not to be merged! (until some fruitful thing is derived)
- only for code storing purposes

### CODE REVIEW

File: add-review.txt [hunk 1]
```diff
@@ -0,0 +1,51 @@
+-- ds.page: Service API call using `http` processor
```
Comment (AbrarNitk) [inline add-review.txt]: @AviralVerma13 why extension is `.txt`?

File: dj/movies/views.py [hunk 2]
```diff
@@ -164,3 +164,38 @@ def add_review(req: django.http.HttpRequest):
 # for a user
 
 # Optional Query. Sort: [release-date, rating, updated_on]
+
+@csrf_exempt
+def get_review(req: django.http.HttpRequest):
+
+    movie_id = req.GET.get("title")
```
Comment (AbrarNitk) [inline dj/movies/views.py:171]: `get_review` API will never provide `title`.

File: dj/movies/views.py [hunk 3]
```diff
@@ -164,3 +164,38 @@ def add_review(req: django.http.HttpRequest):
...
+    print(movie_id)
+    try:
+        review = Review.objects.get(id=movie_id)
+        print("Movie reviews", review)
+        return django.http.JsonResponse(
```
Comment (AbrarNitk) [inline dj/movies/views.py:176]: The response should be a list of `reviews`.

File: add-review.txt (hunk 1 above)
Comment (AviralVerma13) [reply add-review.txt]: that was just a scratch file storing testing movie-review page ui.

**Comment by AviralVerma13** [comment]
work already done through another PR:  https://github.com/FifthTry/movie-db/pull/30

**Pull request closed without merging**